src/native/fs/fs_napi.cpp

```diff
@@ -30,13 +32,68 @@
     #include <sys/statfs.h>
 #endif
 
+// #define GPFS_ATTRFLAG_INCL_ENCR          0x0040  /* Include encryption attributes */
+// #define GPFS_DOT_ENCRYPTION_EA "Encryption"
+
+// typedef struct gpfs_fssnap_handle gpfs_fssnap_handle_t;
+// typedef struct gpfs_ifile gpfs_ifile_t;
+// typedef unsigned int gpfs_ino_t;
+// typedef struct gpfs_iattr gpfs_iattr_t;
+
+// typedef int gpfs_file_t;
+// typedef struct gpfsFcntlHeader gpfsFcntlHeader_t;
+// typedef struct gpfsGetSetXAttr gpfsGetSetXAttr_t;
+
+#define ROUNDUP(X,Y) ( (Y) * ( ((X)+(Y)-1) / (Y) ) )
```

Comment:
I think we don't need most of this defines - probably even all of them. maybe relevant for the encryption I will push in the future

Reply:
done, I started from you code and hoped that you will use it when you finish encryption